lib/layout/box_layout.dart

```diff
@@ -202,7 +202,8 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
       List<Widget> updatedChildren = [];
       for (var i=0; i<children.length; i++) {
         updatedChildren.add(children[i]);
-        if (i != children.length-1) {
+        final child = children[i] as BoxLayout;
+        if (i != children.length - 1 && child.controller.visible) {
```

Comment:
bool visibleChild = child is HasController && (child as HasController).controller.visible

Reply:
Thanks, @vusters I tried your solution but the controller is of type ```Controller```. So I just downcasted to ```WidgetController``` to access the visible bool property

```
final visibleChild = children[i] is HasController && ((children[i] as HasController).controller as WidgetController).visible;
```

I just pushed the code, can you review it?